crates/krun-server/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+name = "krun-server"
+version = { workspace = true }
+authors = { workspace = true }
+edition = { workspace = true }
+rust-version = { workspace = true }
+description = { workspace = true }
+repository = { workspace = true }
+license = { workspace = true }
+
+[dependencies]
+anyhow = { workspace = true, features = ["std"] }
+bpaf = { workspace = true, features = [] }
+env_logger = { workspace = true, features = ["auto-color", "humantime", "unstable-kv"] }
+log = { workspace = true, features = ["kv"] }
+nix = { workspace = true, features = ["user"] }
+rustix = { workspace = true, features = ["fs", "mount", "std", "system", "use-libc-auxv"] }
+tempfile = { workspace = true, features = [] }
+utils = { workspace = true, features = [] }
+vsock = "0.5.0"
+serde = { version = "1.0", features = ["derive"] }
+serde_json = "1.0"
+
+[features]
+default = []
```

Comment:
Missing newline at end of file. I'd recommend setting up EditorConfig for your code editor to avoid this in the future...